.github/workflows/test-release.yml

```diff
@@ -58,6 +60,6 @@ jobs:
         uses: ./.github/workflows/install
 
       - name: Release
-        run: pnpm publish --access public --no-git-checks
+        run: pnpm publish --access public --no-git-checks --provenance
```

Comment:
It seems like that pnpm has not implemented this option for now https://github.com/pnpm/pnpm/pull/6436

I think we can just use npm for now. Once pnpm releases its new version, we can have a seperate PR to upgrade pnpm and change the publish script

Reply:
Sure!